### ISSUE TITLE
No Appimage for the latest release 5.6.0

### ISSUE THREAD

**Issue by personaaleatoria** [ONLYOFFICE/DesktopEditors#430]
Not a feature request or a bug.

I noticed the latest release does not include flatpak or appimage files. I've checked github release page and the official website. Please add options for those of us that use portable or containerized apps on Linux. Thanks for the great software.

**Comment by XDmitryK** [comment]
@personaaleatoria 
Yes, it is planned, but so far we have found several critical bugs that prevent the release
